### PR TITLE
Des 881

### PR DESCRIPTION
**Pull request by JasonLantz**
fix warning in dropdown.....

clean up other warnings from docs.....

### CODE REVIEW

File: demo/demoPages/styles/StylesButtonPage.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const StylesButtonPage = () => (
       <button type="button" className="pe-link--btn" style={marginFour}>Link button 2.0</button>
       <button type="button" className="pe-link--btn" disabled style={marginFour}>Disabled link button 2.0</button>
     <p className="code">
-      {`<div class="pe-btn" tabindex="0" role="button">Div</div>`} <br/>
```
Comment (mhomolak) [inline demo/demoPages/styles/StylesButtonPage.js]: This is showing the example html and is just a string.  That is why the camel case is not used here.

File: .babelrc [hunk 2]
```diff
@@ -5,6 +5,5 @@
     "stage-0"
   ],
   "plugins": [
-    "transform-runtime"
```
Comment (mhomolak) [inline .babelrc]: I'm pretty sure this is required to fix some sort of bug that Jon was encountering in IE relating to phone number.

Comment (wyseguyonline) [reply .babelrc]: Confirmed.

File: demo/demoPages/styles/StylesButtonPage.js (hunk 1 above)
Comment (JasonLantz) [reply demo/demoPages/styles/StylesButtonPage.js]: it should match up with the code....

File: .babelrc (hunk 2 above)
Comment (JasonLantz) [reply .babelrc]: what was the error?

Comment (wyseguyonline) [reply .babelrc]: IIRC, it was an issue with a polyfill for IE not working correctly.

Comment (JasonLantz) [reply .babelrc]: gotcha, thanks...

File: demo/demoPages/styles/StylesButtonPage.js (hunk 1 above)
Comment (mhomolak) [reply demo/demoPages/styles/StylesButtonPage.js]: I understand what you're saying but this is not a React component so I left it in the html format if someone were going to be copy pasting.

**Pull request closed without merging**